waterlog/src/Stylesheets/_map.scss

```diff
@@ -116,13 +119,14 @@
 }
 
 .icon {
-	width: 30px;
-	height: 30px;
-	margin: 2px 2px;
+  width: 35px;
+  height: 35x;
+  margin: 2px 2px;
 }
 
 .icon-home {
```

Comment:
`icon` and `icon-home` is the same class

Reply:
deleted icon-home